iotedgedev/module.py

```diff
@@ -0,0 +1,40 @@
+
+import os
+import json
+
+class Module(object):
+    def __init__(self, output, utility, module_json_file):
+        self.output = output
+        self.utility = utility
+        self.module_json_file = module_json_file
+
+        self.module_language = "csharp"
+        self.file_json_content = None
+        self.load_module_json()
+
+    def load_module_json(self):

```

Comment:
Actually, if module.json not found or there's an error, display a message that says 
not found: "module.json file is required in the root of your module folder."
error: "error while loading module.json file: {path}"  (use self.output.error())

Reply:
With this update we are not requiring module.json file...

Reply:
I meant to say we **are** requiring the module.json file.

Reply:
yes , and I added sys.exit() after logging error message if the file was missing.(Module.py line #32)